packages/react-components/src/molecules/SearchField.tsx

```diff
@@ -59,6 +59,7 @@ const SearchField: React.FC<SearchProps> = (props) => {
     <div css={clearButtonStyles}>
       <TextField
         {...props}
+        maxLength={512}
```

Comment:
Why are we changing the max length of every text field?

Reply:
This does not effect all text fields. This effects all search fields to keep things consistent and has been approved by Beatriz

Reply:
Can we abstract the 512 into a var? 
Then, we can remember why are we using 512 instead of any other number

Reply:
I have made this change